prepare-chroot-base

```diff
@@ -56,9 +56,20 @@ if ! [ -f "${INSTALLDIR}/tmp/.prepared_base" ]; then
 
     mkdir -p "${DOWNLOADDIR}"
     yumconf=$(mktemp)
-    sed -e "s/\\\$releasever/${DIST#fc}/g" \
-        < "${PLUGIN_DIR}"/yum-bootstrap.conf \
-        > "$yumconf"
+
+    # For defined mirroris in builder.conf we need to delete the metalink option
+    # in the repo file because baseurl seems to not override the metalink
+    if [ "x${FEDORA_MIRRORS}" != "x" ]; then
+        sed -e "s/\\\$releasever/${DIST#fc}/g" \
+            -e "/metalink/d" \
```

Comment:
Better comment out instead of removing. And match with `^metalink` to not comment out already commented line.